src/ui/render.types.ts

```diff
@@ -2,3 +2,13 @@ export type CellIdInfo = {
   keyParts: string[];
   isIndex: boolean;
 };
+
+export type SelectionContainer = {
+  selectionStart: Coordinate | null;
+  selectionEnd: Coordinate | null;
+};
+
+export type Coordinate = {
+  rowPosition: number;
+  columnPosition: number;
+};
```

Comment:
This type is also included in #85: https://github.com/lightsheet-team/lightsheet/blob/3744978301b26a664384da32bc07ea4f3fc7e458/src/core/event/events.types.ts#L3-L6
We should use the same type once both PRs are merged.